perseo-main/pom.xml

```diff
@@ -0,0 +1,64 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Licences header in all new files, please (copy paste from a similar existing one).

Reply:
Fixed in [9396d00](https://github.com/telefonicaid/perseo-core/pull/143/commits/9396d0097ab22859d06b72f3f02772efb1065699)